src/Response.php

```diff
@@ -138,7 +146,9 @@ class Response extends AbstractMessage implements ResponseInterface
         428 => 'Precondition Required',
         429 => 'Too Many Requests',
         431 => 'Request Header Fields Too Large',
+        444 => 'Connection Closed Without Response',
         451 => 'Unavailable For Legal Reasons',
+        499 => 'Client Closed Request',
```

Comment:
Doesn't appear to be "official"? Not sure what the stance is on this.
https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#nginx